### ISSUE TITLE
Some fonts have incomplete bracket display in gutter

### ISSUE THREAD

**Issue by MisLink** [CoenraadS/BracketPair#111]
I use font named [Sarasa-Gothic](https://github.com/be5invis/Sarasa-Gothic),  it looks lose the top half in gutter.
![image](https://user-images.githubusercontent.com/8280169/38803500-05a203fa-41a2-11e8-84db-3a80595c80be.png)
I found the code that render brackets in gutter and I try to increase the height:

![image](https://user-images.githubusercontent.com/8280169/38803637-6a0a4b18-41a2-11e8-87c0-b725acd38be4.png)

It all show up.

**Comment by CoenraadS** [comment]
Do you use custom font size or line height also?

**Comment by MisLink** [comment]
It is my font config:
```json
"editor.fontFamily": "Sarasa Mono SC",
"editor.fontSize": 15,
```
I did not change the default line height.

**Comment by WizardPC** [comment]
With FiraCode `{ }` seem to be in intalic.

**Comment by CoenraadS** [comment]
I think I have a fix ready in v1.047. If you have vscode-insiders you can try it out today, else wait for the next vscode release which should be soon.

**Comment by MisLink** [comment]
I tried vscode insider and it's fixed. Thanks for your work!
